src/metabase/email/messages.clj

```diff
@@ -250,13 +250,19 @@
    :content-type "image/png"
    :content      url})
 
+(defn- pulse-link
+  [{:keys [cards]}]
+  (when-let [dashboard-id (some :dashboard_id cards)]
+    {:pulseLink (url/dashboard-url dashboard-id)}))
```

Comment:
Pulses have a `:dashboard_id` column now... can't you use that instead of taking it from `cards`? 

I don't see a situation where it would be different from the IDs in Cards, but if seems weird not to use it